rif.config.js

```diff
@@ -1,18 +1,18 @@
 module.exports = {
   rns: {
     contracts: {
-      rns: '',
-      registrar: '',
-      reverseRegistrar: '',
-      publicResolver: '',
-      nameResolver: '',
-      multiChainResolver: '',
-      rif: '',
-      fifsRegistrar: '',
-      fifsAddrRegistrar: '',
-      rskOwner: '',
-      renewer: '',
-      stringResolver: '',
+      rns: "0x609E5a5606885cFC7d7927523A8a45515d8c69f5",
```

Comment:
Remove this, remember this file should be clean, anyway i know that the .gitignore is failing on skip this file, we should take a look at that too.

Reply:
When the PR is ready i'll delete this, i take note of this for now, so i dont have to be carefully looking to no upload this.

Reply:
@marcosmartinez7 the .env can't be done because it doesn't work properly with extensions, nifty doesn't have any config file, it has js files with config all over the place, we should discuss how to solve this properly.

Reply:
I created a ticket to differentiate between Mainnet and Testnet, also to not show Rif Tools when others Non-Rsk chains are selected